ext/php5/serializer.c

```diff
@@ -17,6 +19,11 @@
 
 ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
 
+#define MAX_ID_LEN 20  // 1.8e^19 = 20 chars
```

Comment:
TBH, this is not defensive, it's easy to forget a + 1 when allocating the buffer and it does no harm to have one byte too much allocated on the stack, better do 21 here and call it maybe `MAX_ID_BUFSIZ`

Reply:
Very good point @bwoebi. I'll have to change my go-to way of doing this from now on. :)

Reply:
Like it. Done